migrate.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
```

Comment:
Nit: this doesn't run on my NixOS system without
```suggestion
#!/usr/bin/env bash
```